tests/unit/s2n_client_hello_retry_test.c

```diff
@@ -586,9 +592,13 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_cert_chain_and_key_free(tls13_chain_and_key));
     }
 
-    /* Test s2n_hello_retry_validate raises a S2N_ERR_INVALID_HELLO_RETRY error when
-     * when conn->handshake_params.server_random is not set to the correct hello retry random value
-     * specified in the RFC: https://tools.ietf.org/html/rfc8446#section-4.1.3 */
+    /*
+     *= https://tools.ietf.org/rfc/rfc8446#4.1.3
+     *= type=test
+     *# Upon receiving a message with type server_hello, implementations MUST
+     *# first examine the Random value and, if it matches this value, process
+     *# it as described in Section 4.1.4).
+     */
```

Comment:
Can you remove the link to the RFC on line 605? It's not needed anymore.

Reply:
Fixed: efaedd1